src/data/graph-data.js

```diff
@@ -79,6 +81,49 @@ async function getLinkedIssues(
   });
 }
 
+export async function GetAllOrganizations(
```

Comment:
Please change the initial to lower case :)

Reply:
Updated, thanks!